nipype/interfaces/ants/utils.py

```diff
@@ -99,13 +104,46 @@ class ImageMath(ANTSCommand, CopyHeaderInterface):
     ...     op2='0.005 0.999 256').cmdline
     'ImageMath 3 structural_maths.nii TruncateImageIntensity structural.nii 0.005 0.999 256'
 
+    >>> pad = ImageMath(
+    ...     op1='structural.nii',
+    ...     operation='PadImage',
+    ...     op2='0.005 0.999 256')
+    >>> pad.inputs.copy_header
+    False
+
```

Comment:
Can you add some text explaining why you're showing these?

Reply:
Added, please let me know if this looks good now.